radicle-httpd/src/api/auth.rs

```diff
@@ -1,13 +1,10 @@
-use std::convert::TryFrom;
 use std::str::FromStr;
 
-use ethers_core::types::{Signature, H160};
+use ethers_core::types::Signature;
```

Comment:
We shouldn't be using ethereum signatures :thinking: 

`ethers_core` dependency can be removed as well.

Reply:
✅